### DISCUSSION TITLE
Option to display duration in minutes

### DISCUSSION THREAD

**Discussion by pmagwene** [jotaen/klog#363] [Feature ideas & discussions]
A nice (hopefully straightforward) addition would be an option to generate summaries where times are reported in total minutes (rather than HH:MM format)

**Comment by jotaen** [comment]
Good idea. I agree it should be fairly straightforward to add. I imagine something like a `--minutes` flag for that, so that you can say `klog total --minutes file.klg` or `klog report --minutes file.klg`.

This is also related to https://github.com/jotaen/klog/issues/189, which is about complementing the `klog print` output with the evaluated duration. A `--minute` flag could then apply likewise there.

A tip for the meantime, in case someone desperately needs this: `klog json` outputs the total in minutes. It’s buried in JSON syntax foo, but [`jq`](https://stedolan.github.io/jq/) could come to the rescue.

**Comment by pmagwene** [comment]
@jotaen Thanks for implementing this!
